cmd/bb_scheduler/BUILD.bazel

```diff
@@ -70,16 +74,37 @@ purgecss_bin.purgecss_binary(
 )
 
 # Create a copy of Bootstrap that only contains the style attributes
-# used by the HTML templates.
-genrule(
-    name = "stylesheet",
+# used by the HTML template.
+js_run_binary(
+    name = "stylesheet_platform",
     srcs = [
         "@com_github_twbs_bootstrap//:css/bootstrap.min.css",
-        ":templates",
```

Comment:
What's the reason we can't continue to use the filegroup here? Why do we need to pass in the results of `glob()` directly?

Reply:
Because we can't use `locations` as we need to prefix each item in the filegroup by `../../../`. And we can only use `location` on files that are specifically mentioned in `srcs`, so even though the filegroup `templates` includes all of the items we want to call `location` on, we get an error if we do that.

Reply:
Gotcha! Am I correct that the `filegroup(name = "templates")` target is now unused? If so, let's remove it.

Reply:
Ah yes. That lint check assumes that there's always a target that has the same name as what's used in `//go:embed`. Yeah, then we should leave that.